crates/sui-core/src/generate_format.rs

```diff
@@ -146,8 +142,6 @@ fn get_registry() -> Result<Registry> {
     tracer.trace_type::<TypedStoreError>(&samples)?;
     tracer.trace_type::<ObjectInfoRequestKind>(&samples)?;
     tracer.trace_type::<TransactionKind>(&samples)?;
-    tracer.trace_type::<MoveStructLayout>(&samples)?;
```

Comment:
I removed these since I don't understand why we ever kept them as there are custom deserializers for these. But please double-check me and make sure this is OK to remove!

Reply:
these should be fine to remove, afaik but we could always add something back here if need be. None of these should be a part of our public api